### DISCUSSION TITLE
Needing help with a ban

### DISCUSSION THREAD

**Discussion by ghost** [community/community#84061] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

So not too long ago my original github account https://github.com/GhostyTongue was banned out of no where without a ban reason And I sat there devastated for a few weeks and I want it to get unbanned.. I read tos and I didn't break any tos rules

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Ghosty-Tongue ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
